kuma/search/tests/test_serializers.py

```diff
@@ -59,6 +59,25 @@ def test_document_serializer(self):
         assert isinstance(dict_data, dict)
         assert dict_data['id'] == result[0].id
 
+    def test_search_query_serializer(self):
+        search_serializer = SearchQuerySerializer(
+            data={'q': 'test'}
+        )
+        self.assertTrue(search_serializer.is_valid())
+        assert {} == search_serializer.errors
+
+        search_serializer = SearchQuerySerializer(
+            data={'q': 'test\\nsomething'}
+        )
+        self.assertFalse(search_serializer.is_valid())
+        self.assertEqual(set(search_serializer.errors.keys()), set(['q']))
```

Comment:
```suggestion
        self.assertEqual(list(search_serializer.errors.keys()), ['q'])
```
Sets are useful when you want to compare two arrays/lists/sequences independent of order. But when there's only exactly 1 it's not necessary.